### DISCUSSION TITLE
Translation for the key XXX is missing

### DISCUSSION THREAD

**Discussion by JanMann89** [stephanrauh/ngx-extended-pdf-viewer#1604] [Q&A] [answered]
Hi, when I load the viewer I get a bunch of warnings in my console, that say that the translation files could not be found:

![image](https://user-images.githubusercontent.com/51460788/205033603-084221e8-b41e-40fe-8457-1a0f2b7e4580.png)

I currently use version `15.0.12` and Angular `13.3.11`.

When I build my project the files are as expected in my assets folder: `/assets/locale/` contains 109 subfolders and the file `local.properties`.

I initialized the component as it follows:
```
<ngx-extended-pdf-viewer
  *ngIf="pdfPath"
  [filenameForDownload]="pdfFilename"
  [src]="pdfPath"
  [useBrowserLocale]="true"
  [ignoreKeyboard]="true"
  [handTool]="false"
  [sidebarVisible]="false"
  [textLayer]="true"
  [customToolbar]="multiToolbar"
>
```

In my app.module I registered my location:
```
registerLocaleData(localeDe, 'de-DE', localeDeExtra);
```

Is there a solution to fix this? Thanks in advance!

**Comment by stephanrauh** [comment, marked as answer]
Please update to version 15.1.0. You've run into a side-effect of an improvement I've implemented with version 15.0.12. I've examined the problem thoroughly and eliminated the need to add the attribute `[useBrowserLocale]`. 

That's why I've incremented the minor version number. Otherwise, version 15.1.0 is basically a bug-fix release. You can update to it without having to run a lot of tests.

**Reply by JanMann89** [reply]
Perfect, thanks a lot for the super fast update! It seems to work. Unfortunately there is a leftover debugging snippet, but all in all it solved my problem.

https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/main/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.component.ts#L1118

**Reply by stephanrauh** [reply]
Thanks for pointing this out! I guess I need to add a check for forgotten `debugger` statements to my `npm run release` script.
